test/integration/gc_default_test.go

```diff
@@ -83,7 +83,7 @@ func TestGCDefaults(t *testing.T) {
 
 	// the /oapi endpoints should orphan by default
 	// wait for a bit and make sure that the build is still there
-	time.Sleep(2 * time.Second)
+	time.Sleep(6 * time.Second)
```

Comment:
All the sleeps in this test smell bad -- not clear they are going to remain valid in CI with this much parallelism... @deads2k can we fix

Reply:
> Hi guys, Maybe using sync.WaitGroup you could do that without time.Sleep, if I'm not wrong.

There's nothing to be notified by. There isn't any visibility into the GC controller cache.